.travis.yml

```diff
@@ -14,16 +14,15 @@ before_install:
  - sudo bash -c "mount -t hugetlbfs nodev /mnt/huge"
  - sudo apt-get update
 
- - cd ~
- - git clone http://dpdk.org/git/dpdk
- - cd dpdk
- - git checkout -b v19.02 v19.02
- - export RTE_SDK=~/dpdk
- - make config T=x86_64-native-linuxapp-clang
- - sed -i -e "s/CONFIG_RTE_LIBRTE_PMD_PCAP=n/CONFIG_RTE_LIBRTE_PMD_PCAP=y/" build/.config
- - make EXTRA_CFLAGS='-fPIC'
-
 script:
+ - mkdir -p ~/dpdk
+ - mv ./dpdk.tar.bz2 ~/dpdk
+ - cd ~/dpdk
+ - tar -xvf dpdk.tar.bz2
+ - ls ~
+ - ls ~/dpdk
+ - ls ~/dpdk/build
```

Comment:
what ? Why did you use `ls` in this script ?
Also, you would prefer clone/copy dpdk in `$TRAVIS_BUILD_DIR/dpdk` instead of `~/dpdk`

Reply:
ls was for testing, should be removed in the end